lib/core/extension/proposal_model_extensions.dart

```diff
@@ -6,8 +6,12 @@ extension ProposalModelTimeFormatting on ProposalModel {
 
     final now = DateTime.now();
     final expirationDate = expiration!.toLocal();
-    final difference = expirationDate.difference(now);
 
+    if (expirationDate.isBefore(now)) {
+      return 'Expired';
```

Comment:
Not sure this is the best term - is this UX visible?

It should be the same as on the web... maybe passed of failed?